creator-node/src/services/sync/syncQueue.ts

```diff
@@ -1,36 +1,57 @@
-const { Queue, Worker } = require('bullmq')
+import type { SpanContext } from '@opentelemetry/api'
 
-const { clusterUtils } = require('../../utils')
-const { instrumentTracing, tracing } = require('../../tracer')
-const {
+import { Job, Queue, Worker } from 'bullmq'
+import { Redis } from 'ioredis'
+
+import { clusterUtils } from '../../utils'
+import { instrumentTracing, tracing } from '../../tracer'
+import {
   logger,
   logInfoWithDuration,
   logErrorWithDuration,
   getStartTime
-} = require('../../logging')
-const secondarySyncFromPrimary = require('./secondarySyncFromPrimary')
+} from '../../logging'
+import secondarySyncFromPrimary from './secondarySyncFromPrimary'
+import { ForceResyncConfig } from '../../services/stateMachineManager/stateReconciliation/types'
 
 const SYNC_QUEUE_HISTORY = 500
 
+type EnqueueSyncArgs = {
+  wallet: string
+  creatorNodeEndpoint: string
+  blockNumber: number
+  forceResyncConfig: ForceResyncConfig
+  forceWipe?: boolean
+  logContext: Object
+  parentSpanContext: SpanContext
+}
+
 /**
  * SyncQueue - handles enqueuing and processing of Sync jobs on secondary
  * sync job = this node (secondary) will sync data for a user from their primary
  */
-class SyncQueue {
+export class SyncQueue {
```

Comment:
One day we'll type `config.js` and we won't have to use `any` .... but that day is not today